packages/js/components/src/experimental-select-control/menu.tsx

```diff
@@ -22,9 +29,25 @@ export const Menu = ( {
 	isOpen,
 	className,
 }: MenuProps ) => {
+	const [ boundingRect, setBoundingRect ] = useState< DOMRect >();
+	const selectControlMenuRef = useRef< HTMLDivElement >( null );
+
+	useEffect( () => {
+		if (
+			selectControlMenuRef.current &&
```

Comment:
You can get rid of this, right? the `selectControlMenuRef.current?.parentElement` accounts for this with the conditional chaining.

Reply:
Done as part of https://github.com/woocommerce/woocommerce/pull/34967/commits/8e895d4139b8369fd8c90943818bb20c7d10cf06